.github/actions/label-and-move-released-issues-v1/src/run.test.ts

```diff
@@ -132,12 +133,16 @@ describe('run', () => {
     const projectNumber = getInput
       .withArgs('project-number')
       .returns(inputs?.projectNumber ?? '66')
+    const releaseColumn = getInput
```

Comment:
Just missing two tests here: 

- When the input `column-name` is supplied ensure we respect that e.g. `some-other-column-name`
- When the inout is not provided ensure we default to `released`

Reply:
as discussed with @Zidious this is not needed